src/status_im2/contexts/chat/bottom_sheet_composer/view.cljs

```diff
@@ -21,152 +21,164 @@
+                                                   constants/input-height)
+        animations                               {:gradient-opacity  (reanimated/use-shared-value 0)
+                                                  :container-opacity (reanimated/use-shared-value
+                                                                      (if (utils/empty-input?
+                                                                           input-text
+                                                                           images
+                                                                           reply
+                                                                           audio)
+                                                                        0.7
+                                                                        1))
+                                                  :height            (reanimated/use-shared-value
+                                                                      initial-height)
+                                                  :saved-height      (reanimated/use-shared-value
+                                                                      initial-height)
+                                                  :last-height       (reanimated/use-shared-value
+                                                                      (utils/bounded-val
+                                                                       @content-height
+                                                                       constants/input-height
+                                                                       max-height))
+                                                  :opacity           opacity
+                                                  :background-y      background-y}
+        dimensions                               {:content-height content-height
+                                                  :max-height     max-height
+                                                  :window-height  window-height
+                                                  :lines          lines
+                                                  :max-lines      max-lines}
+        show-bottom-gradient?                    (utils/show-bottom-gradient? state dimensions)
+        cursor-pos                               (utils/cursor-y-position-relative-to-container props
+                                                                                                state)]
+    (effects/initialize props
+                        state
+                        animations
+                        dimensions
+                        chat-input
+                        keyboard-height
+                        (boolean (seq images))
+                        reply
+                        edit
+                        audio)
+    (effects/setup-selection props)
+    (utils/update-input props state input-text)
```

Comment:
whats that and why it's in utils, it's not an effect?

Reply:
Fixed it